tower_eval/cli.py

```diff
@@ -154,6 +157,11 @@ def run_evaluations(configs: dict) -> dict:
                     eval_args = combine_metrics_args(
```

Comment:
This will take care of reseting the eval args to the defaults of the task metric, right?
I mean if we have only a single subtask with specific args in the middle of all the subtasks once we are done with that subtask, we will get back to the default values. is that correct?

For example, in the config below, what will be the arguments for evaluating `booking_tickets-dynamic.en-fr` and `classpass_chat-dynamic.en-de`?

```
tasks:
  - name: mt
    subtasks:
      aboutyou-ticket-dynamic.en-de:
      am_eagle_chat-dynamic.en-fr:
      apple_apple_chat_for_core_chat-dynamic.en-it:
      belkin_chat-dynamic.en-ko:
        eval_args:
          metrics:
            bleu:
              tokenizer: ko-mecab
      belkin_chat-dynamic.en-zh-CN:
        eval_args:
          metrics:
            bleu:
              tokenizer: zh
      booking_tickets-dynamic.en-fr:
      classpass_chat-dynamic.en-de:
```

Reply:
yes, it's the same function as before

Reply:
also fixed a bug where metric was being reinstantiated if there were subtask args, but not reinstantiated again to reset back to the task args